kuma/wiki/jinja2/wiki/document.html

```diff
@@ -25,9 +25,9 @@
   {% set approvals_html = get_approvals_html(document, request.user) %}
 {% endif %}
 
-{% set zone_stack = document.zone_stack %}
+{% set zone_stack = document.zone_stack or document.default_zone_stack %}
 {% set is_zone = zone_stack|length %}
-{% set is_zone_root = is_zone and zone_stack[0].document == document %}
+{% set is_zone_root = is_zone and (zone_stack[-1].document in (document, document.parent)) %}
```

Comment:
Thanks @jwhitlock. I think I understand now. I was interpreting the meaning of the zone root as the document being associated with the topmost zone, rather than the nearest zone.